binder/summary-statistics.ipynb

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
name this file reconstruction-quality-summary-statistics.ipynb

Reply:
Renamed in 4d621cc